File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dict = {}
        for i in range(len(nums)):
            dict[nums[i]] = i

        for i in range(len(nums)):
            j = dict.get(target - nums[i])
            if None == j:
                continue

            if i != j:
                return([i, j])

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (target == nums[i] + nums[j]) {
                    /*
                    int[] pair = new int[2];
                    pair[0] = i;
                    pair[1] = j;
                    return pair;
                    */
                    int[] pair = {i, j};
                    return pair;
                }
            }
        }

        return null;
    }
}